packages/manager/src/components/ActivityStream/ActivityStream.tsx

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable no-underscore-dangle */
+/* eslint-disable */
```

Comment:
Remove all just`eslint-disable` written over the files.

This is making the eslint skipping these files. If u wanna disable only some rules like no-underscore-danger kindly disable that one. All other rules kindly follow it. 

After removing eslint disable, run eslint and fix up those one. Then ill review this again

Ref: https://eslint.org/docs/latest/use/configure/rules#disabling-rules

Reply:
Sure, will do this and again push for review